assets/src/apps/authoring/components/ComponentToolbar/AddComponentToolbar.tsx

```diff
@@ -111,9 +112,10 @@ const AddComponentToolbar: React.FC<{
     dispatch(setCurrentSelection({ selection: newPartData.id }));
 
     dispatch(setRightPanelActiveTab({ rightPanelActiveTab: RightPanelTabs.COMPONENT }));
+    dispatch(setCopiedPart({ copiedPart: null }));
```

Comment:
Once a copied component is pasted, we need to clear the selection otherwise it always tries to paste the part when you do something else.